cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -102,16 +102,6 @@ public void run() {
                 editBox.setTextColor(Color.WHITE);
                 editBox.setSingleLine();
                 editBox.setOpenGLViewScaleX(scaleX);
-                Resources r = mCocos2dxActivity.getResources();
```

Comment:
yeah, it's better to do this. And I think we shouldn't change the anchor point of the label and the placeholder label.